### DISCUSSION TITLE
Jupyter book vs MyST-tools

### DISCUSSION THREAD

**Discussion by fpopineau** [executablebooks/meta#1021] [General] [answered]
Hi,
I am new here, discovering both jupyter-book and mystjs. 
It appears that both projects allow to build an HTML website
from MD files (with extensions) and IPYNB notebooks.
But how do they relate one to the other?
There is the myst.yml configuration file for myst-tools
and the ._config.yml file for jupyter-book that 
seem to be stepping on each other's toes?
Thanks for any help to see more clearly through it.

**Comment by ccalvocm** [comment]


**Reply by rossbar** [reply]
FWIW this reads like chatgpt-generated boilerplate that doesn't really address the original question IMO.

**Comment by rowanc1** [comment, marked as answer]
Hi @fpopineau there is a bit of background on the `mystjs` project and how it relates to JupyterBook here:
https://myst-tools.org/docs/mystjs/background

There is increasing overlap in the projects as we are using the mystjs project to create live previews of JupyterBooks/myst pages in JupyterLab through the [jupyterlab-myst](https://github.com/executablebooks/jupyterlab-myst) plugin.

There are still a number of things that you cannot do with MySTjs, as it relates to JupyterBook, which this issue is tracking: https://github.com/executablebooks/mystjs/issues/189. The mystjs project has more of a focus on scientific writing, papers, PDF/latex generation than JupyterBook - there are some tutorials here:
https://myst-tools.org/docs/mystjs/quickstart-myst-documents

JupyterBook is based on Sphinx (a documentation engine) which has the `_config.yml` to supply sphinx extensions and other options for JupyterBook. The `myst.yml` serves a similar role for the mystjs stack, but is currently not compatible. We expect to improve compatibility in the coming months!

Many people are using the tools together through the Jupyter extension and/or to export PDF articles of their JupyterBook content. For choosing between the projects: (1) note that the mystjs is still in beta, and has a few rough edges and limited extensibility; (2) if you are looking to do any Python documentation as a part of your book, the Sphinx/JupyterBook stack is a better choice; (3) if you are looking to write scientific papers with notebooks -- mystjs has more related features/latex templates/export options than JupyterBook for that use case.
